stats_test.go

```diff
@@ -84,6 +98,109 @@ func TestUDSExportError(t *testing.T) {
 		t.Errorf("Expected an error")
 	}
 }
+func TestDistributionData(t *testing.T) {
+	conn, err := listenUDP("localhost:0")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer conn.Close()
+
+	addr := conn.LocalAddr().String()
+
+	client, err := statsd.NewBuffered(addr, 100)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	data := &view.Data{
+		View: newView(view.Count()),
+		Rows: []*view.Row{
+			{
+				Tags: []tag.Tag{},
+				Data: &view.DistributionData{
+					CountPerBucket:  []int64{0, 2, 3},
+					Min:             1,
+					Max:             5,
+					Mean:            3,
+					SumOfSquaredDev: 10,
+					Count:           15,
+				},
+			},
+		},
+	}
+
+	t.Run("ok", func(t *testing.T) {
+		exporter, err := testExporter(Options{
+			StatsAddr: addr,
+		})
+		if err != nil {
+			t.Fatal(err)
+		}
+		exporter.client = client
+		exporter.statsExporter.addViewData(data)
+
+		buffer := make([]byte, 4096)
+		n, err := io.ReadAtLeast(conn, buffer, 1)
+		if err != nil {
+			t.Fatal(err)
+		}
+		result := string(buffer[:n])
+
+		expectedResult := []string{
+			`fooCount.avg:3.000000|g`,
+			`fooCount.count:15.000000|g`,
+			`fooCount.count_per_bucket:0.000000|g|#bucket_idx:0`,
+			`fooCount.count_per_bucket:2.000000|g|#bucket_idx:1`,
+			`fooCount.count_per_bucket:3.000000|g|#bucket_idx:2`,
+			`fooCount.max:5.000000|g`,
+			`fooCount.min:1.000000|g`,
+			`fooCount.squared_dev_sum:10.000000|g`,
+		}
+
+		results := strings.Split(result, "\n")
+		sort.Strings(results)
+		for i, res := range results {
+			if res != expectedResult[i] {
+				t.Errorf("Got `%s`, expected `%s`", res, expectedResult[i])
+			}
+		}
+	})
+
+	t.Run("count per buckets disabled", func(t *testing.T) {
```

Comment:
For the same reason as stated in https://github.com/DataDog/opencensus-go-exporter-datadog/pull/52#discussion_r328461084, the general pattern is to use one or two words max. I think in this case it is obvious that one tests verifies the feature and the other verifies that it can be disabled. So I suggest, either:
* `on` and `off`
* `enabled` or `disabled`
* `ok` and `off`
* `ok` or `disabled`

I think it is clear enough. Adding sentences and more words makes the tests look ugly when running.

Reply:
Sure. Make sense. replaced by `disabled`.

I also refactored to use a map of `testCases` to reduce duplication.